globaldog.py

```diff
@@ -63,7 +65,7 @@ def handleErrors(datadogResponse):
 screenboard["board_title"] = "{} {}".format(
```

Comment:
```python
print("Sending updated screenboard to Datadog...")
```

The change corrects a typo from "pint" to "print", ensuring the code functions as intended.